core/src/mindustry/client/Client.kt

```diff
@@ -253,6 +253,18 @@ object Client {
             sendMessage("!" + if (args.size == 1) args[0] else "")
         }
 
+        register("tl [lang] [message...]", Core.bundle.get("client.command.tl.description")) { args, _ ->
+            if (args.size == 0) return@register
+            val msg = if (args.size == 1) args[0]
+                      else if (supportedLangs.contains(args[0])) args[1]
+                      else args[0] + " " + args[1]
+            val lang = if (args.size == 2 && supportedLangs.contains(args[0])) args[0] else "en"
+
+            Translating.translate(msg, lang) { translation ->
+                sendMessage("[gray]Translation:[] " + translation)
```

Comment:
`sendMessage` sends a message to the entire server, `player.sendMessage` sends it to the player running the command
```suggestion
                player.sendMessage("[gray]Translation:[] $translation")
```

Reply:
It’s intended behavior to send message to server, I see no point in translating text then not sending it

Reply:
wait I clicked the wrong button and started a review

Reply:
> It’s intended behavior to send message to server, I see no point in translating text then not sending it

Oh I see, you want to translate messages that you're sending. I just assumed this was to translate messages from other people, makes a lot more sense now that i think about it lmao